IPs/RSKIP134.md

```diff
@@ -12,55 +12,55 @@
 
 ## Abstract
 
-RSK has a two way peg system to lock funds in the Federation, using BTC, that are then released as RBTC in RSK through the Bridge native contract. This system was implemented using a whitelisting process that adds a nuisance in the usability of the network.
+RSK has a two-way peg system to lock funds in the Federation, using BTC, which is then released as RBTC in RSK through the Bridge native contract. This system was implemented using a whitelisting process that adds a nuisance to the usability of the network.
 
-We will define a process that replaces the whitelisting by restricting the funds locked in the Federation at any time, taking us closer to a fully decentralized two way peg, while avoiding unnecesary risks for the network users.
+This RSKIP defines a process on the side of the whitelisting to restrict the funds locked in the Federation at any time, taking the platform closer to a fully decentralized two-way peg, while avoiding unnecessary risks for the network users.
```

Comment:
Add a full stop 

...the funds locked in the Federation at any time. Taking the platform closer...